src/wavesurfer.js

```diff
@@ -767,6 +767,15 @@ export default class WaveSurfer extends util.Observer {
         return this.backend.play(start, end);
     }
 
+    /**
+     * Set a point in seconds for playback to stop at.
+     *
+     * @param {number} position Position (in seconds) to stop at
```

Comment:
add a jsdoc version marker for 3.3.0 (also in webaudio backend)

Reply:
done